config/manager/default-monitoring.yaml

```
@@ -76,13 +76,14 @@ data:
             usage: "GAUGE"
             description: "Total number of backends that are currently waiting on other queries"
 
-    pg_database:
+    pg_database_frozen_xid:
       query: |
         SELECT datname
           , pg_catalog.pg_database_size(datname) AS size_bytes
           , pg_catalog.age(datfrozenxid) AS xid_age
           , pg_catalog.mxid_age(datminmxid) AS mxid_age
         FROM pg_catalog.pg_database
+          WHERE datname not in ('template0','template1')
```

I think that you should not exclude `template1` from the check. If it allows connections, it should be checked. I suggest excluding all and only the databases that do not allow connections.
```suggestion
          WHERE datallowconn
```